keras_preprocessing/image/directory_iterator.py

```diff
@@ -68,6 +68,7 @@ def __init__(self,
                  directory,
                  image_data_generator,
                  target_size=(256, 256),
+                 keep_aspect_ratio=False,
```

Comment:
Please put the new argument toward the end of the constructor. After `interpolation` would be good.